src/pyOpenMS/tests/unittests/test_AcquisitionInfo.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+import os
+
+import pyopenms
+class TestAcquisitionInfo(unittest.TestCase):
+
+	def setUp(self):
+		dirname = os.path.dirname(os.path.abspath(__file__))
+		self.filename_mzml = os.path.join(dirname, "test.mzML").encode()
+
+	def test_acquisitioninfomemberaccess(self):
+		exp = pyopenms.MSExperiment()
+		pyopenms.MzMLFile().load(self.filename_mzml, exp)
+
+		# Basically test that the output is non-zero (e.g. the data is
+		# correctly relayed to python)
+		# The functionality is not tested here!
+
+		# starting point
+		self.assertEqual(exp[0].getAcquisitionInfo().size(), 1)
+		self.assertNotEqual(exp[0].getAcquisitionInfo().size(), 0)
+
+		# metainfo
+		exp[0].getAcquisitionInfo().size()  # is 1
+		self.assertEqual(exp[0].getAcquisitionInfo()[0].isMetaEmpty(), True)  # is True
+
+		tmp = exp.getSpectra()
+		tmp[0].getAcquisitionInfo()[0].setMetaValue('key', 420)
```

Comment:
I guess this suffers from the same problem, we get a copy of all acquisition info objects